akka-docs/src/main/paradox/scala/common/binary-compatibility-rules.md

```diff
@@ -135,7 +135,7 @@ manual exception overrides may be put in place if the change happened to be in a
 ## Serialization compatibility across Scala versions
 
 Scala does not maintain serialization compatibility across major versions. This means that if Java serialization is used
-there is no guarantee objects can be cleanly deserialized if serialized with a different version of Scala.
+there is no guarantee objects can be cleanly de-serialized if serialized with a different version of Scala.
```

Comment:
I think we regularly use that as a single word

Reply:
Removed